extensions/cornerstone/src/index.js

```diff
@@ -24,8 +24,8 @@ export default {
    * @param {object} [configuration={}]
    * @param {object|array} [configuration.csToolsConfig] - Passed directly to `initCornerstoneTools`
    */
-  preRegistration({ serviceManager, configuration = {} }) {
-    init({ serviceManager, configuration });
+  preRegistration({ servicesManager, configuration = {} }) {
```

Comment:
I feel like the single most difficult thing in programming is "what should I name this thing?"
I'm okay with `servicesManager` -- I constantly struggle with if things like this should be plural or not.

Reply:
Me too! In this case the manager is handling all services, so I think it's ok to use the plural.